app/Css/Constants.hs

```diff
@@ -455,13 +455,14 @@ nodeFontSize :: Num a => a
 nodeFontSize = 12
 
 -- |Defines hybrid font size, 7 in pixels.
-hybridFontSize :: Num a => a
+hybridFontSize :: Integer a => a
```

Comment:
When I write in the following code:
hybridFontSize :: Num a => a
hybridFontSize = 7
hybridFontSize :: Integer

I get the error:
C:\Users\Frank2\Desktop\Courseography\courseography\app\Css\Constants.hs:460:1: error:
    Duplicate type signatures for `hybridFontSize'
    at app\Css\Constants.hs:458:1-14
       app\Css\Constants.hs:460:1-14

Am I doing this incorrectly David?